scripts/sdk-api-diff.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+usage="--------------------------------------------
+Description: Compares API differences between two module versions.
+
+Usage: sdk-api-diff <module name> <module A path> <module B path> [OPTIONS]
+
+Options:
+  --target: Target platform and version e.g. x86_64-apple-ios14.0-simulator or arm64-apple-ios14.0
+    (Default: x86_64-apple-ios14.0-simulator)
+  --sdkpath: Path to target's SDK.
+    (Default: output of 'xcrun --sdk iphonesimulator --show-sdk-path')"
+
+# default values
+TARGET_NAME=x86_64-apple-ios14.0-simulator
+SDK_DIR=`xcrun --sdk iphonesimulator --show-sdk-path`
+
+for i in "$@"; do
+  case $i in
+    --target=*)
+      TARGET_NAME="${i#*=}"
+      shift
+      ;;
+    --sdkpath=*)
+      SDK_DIR="${i#*=}"
+      shift
+      ;;
+    --help)
+      printf -- "$usage"
+      exit 0
+      ;;
+    *)
+      # unknown option
+      ;;
+  esac
+done
+
+if [ -z ${1+x} ] || [ -z ${2+x} ] || [ -z ${3+x} ]; then
+    echo "Error: Missing required arguments"
+    printf -- "$usage"
+    exit 1
+fi
+if [ ! -d $2 ]; then
+    echo "Invalid path: $2"
+    exit 1
+fi
+if [ ! -d $3 ]; then
+    echo "Invalid path: $3"
+    exit 1
+fi
+REQUEST_TEMPALTE="sdk-api-diff-request-tempalte.yml"
+if [ ! -f $REQUEST_TEMPALTE ]; then
+    echo "Missing $REQUEST_TEMPALTE file"
+    exit 1
+fi
+
+hash jq 2> /dev/null || { echo >&2 "ERROR: jq must be installed"; exit 1; }
+hash sourcekitten 2> /dev/null || { echo >&2 "ERROR: sourcekitten must be installed"; exit 1; }
```

Comment:
Maybe add "sourcekitten comes with Xcode so ensure you have a recent version of Xcode installed on your system"

Reply:
Sourcekitten comes with Xcode???

Reply:
Sourcekitten is an external dependency installed by homebrew (or as a part of jazzy gem) but yeah, I'll add more infor about installation

Reply:
oh, sorry, I was getting this confused with "something" 😕